Library/PledgeViewContext.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+
+public enum PledgeViewContext {
+  case pledge
+  case update
+  case changePaymentMethod
+
+  var confirmationLabelHidden: Bool {
```

Comment:
These booleans basically map directly in `PledgeViewModel`, it just helped for readability to do this here. They're tested in both places.